src/ArcPay.sol

```diff
@@ -51,7 +51,7 @@ contract ArcPay is Initializable, Ownable2StepUpgradeable, UUPSUpgradeable {
 
     uint public mintHashChain;
     uint provedMintTimeStamp = 0;
-    mapping(uint mintHash => uint timestamp) mints;
+    mapping(uint => uint) mints; // minthash => timestamp
```

Comment:
the original is valid solidity. you may be on an older version. try 0.8.19

Reply:
but this is a good catch. we should pin the version in foundry.toml.

Reply:
Ah yup you're right! Reverted that and just updated the pragma statement for this file so that it compiles properly immediately.

Reply:
additionally, can you add `solc_version = '0.8.19'` to foundry.toml to pin the compiler version?